jetstream/cli.py

```diff
@@ -112,10 +154,8 @@ def execute(
             [], ExternalConfigCollection
         ] = ExternalConfigCollection.from_github_repo,
         today: Optional[datetime] = None,
-        strategy: Callable[[str, str], ExecutorStrategy] = SerialExecutorStrategy,
```

Comment:
It feels slightly more convenient to directly pass in the class instance here instead of having to pass in some lambda for defining a different `ExecutorStrategy`

Reply:
:+1: yeah, makes sense